tests/search/range/hash_dictionary/test.sd

```diff
@@ -4,17 +4,16 @@ search test {
      field i1 type int {
          indexing: attribute | summary
          attribute: fast-search
-         match:exact
+         dictionary:hash
      }
      field f1 type int {
          indexing: attribute | summary
          attribute: fast-search
-         match:exact
+         dictionary:hash
      }
      field m1 type array<int> {
          indexing: attribute | summary
          attribute: fast-search
-         match:exact
```

Comment:
Was it intentional to remove this?

Reply:
Yes, that was a temporary hack for triggering the hash.